pkg/monitor/intervalcreation/node.go

```diff
@@ -201,7 +201,10 @@ func readinessFailure(logLine string) monitorapi.Intervals {
 			Condition: monitorapi.Condition{
 				Level:   monitorapi.Info,
 				Locator: containerRef.ToLocator(),
-				Message: monitorapi.ReasonedMessage(monitorapi.ContainerReasonReadinessFailed, message),
+
+				// Add pod and namespace to the message so tests can filter for certain pods and also
+				// ignore messages from those same pods.
+				Message: monitorapi.ReasonedMessage(monitorapi.ContainerReasonReadinessFailed, "pod="+containerRef.Pod.Name, "containerName="+containerRef.ContainerName, failureTime.Format("15:04:05"), message),
 			},
 			From: failureTime,
```

Comment:
doesn't this give you the time?

Reply:
> I want the time in the message so that when we look at the failureoutput in a failed test

want to fix the general output problem?

Reply:
ok, I removed the time from the event message and added the time in the failure output of the actual test where it would be showed in the prow job test failure.